### PR TITLE
Data/Config Upload to IPFS

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#145  and #144 

##### Added
- `get_data` function
- `get_config` function
##### Changed
- `upload_data` parameter added to `nft_storage` method
- `upload_config` parameter added to `nft_storage` method

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -247,6 +247,12 @@ Upload generated image directly to [NFT.storage](https://NFT.storage)
 {'status': True, 'message': 'FILE_LINK'}
 ```
 
+You can also upload your config/data to nft storage as follows:
+```pycon
+>>> g.nft_storage(api_key="API_KEY", upload_config=True)
+{'status': {'image': True, 'config':True}, 'message': {'image':'IMAGE_FILE_LINK', 'config':'CONFIG_FILE_LINK'}
+```
+
```
Comment (sepandhaghighi) [inline README.md:255]: Please add an example for `upload_data` mode

File: examples/demo.ipynb [hunk 2]
```diff
@@ -389,6 +389,22 @@
...
+    "You can also upload your config/data to nft storage as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "g.nft_storage(api_key=\"YOUR_API_KEY\", upload_config=True)"
```
Comment (sepandhaghighi) [inline examples/demo.ipynb:406]: `g1` instead of `g`

File: examples/demo.ipynb [hunk 3]
```diff
@@ -389,6 +389,22 @@
...
+   ]
```
Comment (sepandhaghighi) [inline examples/demo.ipynb:407]: Please add an example for `upload_data` mode

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

**Comment by sadrasabouri** [comment]
@sepandhaghighi can you please review it again? I fixed the problems.

File: test/nft_upload_test.py [hunk 4]
```diff
@@ -20,4 +20,17 @@
 ...     time.sleep(10)
 >>> status
 True
+>>> status = False
+>>> counter = 0
+>>> while(status == False and counter<try_limit):
```
Comment (sepandhaghighi) [inline test/nft_upload_test.py:25]: To increase coverage, I suggest splitting this test:

```pycon
>>> status = False
>>> counter = 0
>>> while(status == False and counter<try_limit):
...     result = g.nft_storage(api_key=NFT_STORAGE_API_KEY, upload_data=True, depth=1)
...     counter = counter + 1
...     status = result["status"]
...     time.sleep(10)
>>> status["image"]
True
>>> status["data"]
True
>>> status = False
>>> counter = 0
>>> while(status == False and counter<try_limit):
...     result = g.nft_storage(api_key=NFT_STORAGE_API_KEY, upload_config=True, depth=1)
...     counter = counter + 1
...     status = result["status"]
...     time.sleep(10)
>>> status["image"]
True
>>> status["config"]
True
```

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

**Comment by sadrasabouri** [comment]
@sepandhaghighi I believe now it's ready. :rocket:

**Review by sepandhaghighi — APPROVED** [review]
LGTM 🚀

**Pull request merged**